chapter2/tests/t2/chapter2-t2-gen-kustomization.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+if [ "$#" -eq 0 ]; then
+  echo "Usage: chapter2-t2-gen-kustomization.sh [REPO]..."
+  exit 1
+fi
+
+readonly DEPLOY_KEY_POSTFIX="deploy-key.pem"
+
+# stop on first error
+set -e
```

Comment:
We also use `set -eo pipefail` - https://gist.github.com/vncsna/64825d5609c146e80de8b1fd623011ca

Also please move it at the beginning of the bash script, so it would start work from the beginning (not from the middle of the file)